browser/src/control/Control.Notebookbar.js

```diff
@@ -327,18 +323,13 @@ L.Control.Notebookbar = L.Control.extend({
 		this.reloadShortcutsBar();
 	},
 
-	showShortcutsButton: function(buttonId, show) {
-		var i = this.hiddenShortcutButtons.indexOf(buttonId);
-		if (i > -1) {
-			if (show === true)
-				this.hiddenShortcutButtons.splice(i, 1);
-
-			this.reloadShortcutsBar();
-			return;
+	showButton: function(buttonId, show) {
+		var button = $('.notebookbar-scroll-wrapper #' + buttonId);
```

Comment:
I think this will break initial functionality: ability to show/hide shortcutsbar icons, shortcut icons are not inside: `.notebookbar-scroll-wrapper`

![scroll wrapper](https://github.com/CollaboraOnline/online/assets/5307369/295ed9d6-d9ff-44aa-bdf1-c8e3db8a32eb)

Reply:
I think there is some confusion here. This is only supposed to show/hide the notebook bar button. `showShortcutsButton` is removed because it no longer need to handle `hiddenShortcutButtons` and it is called in one place, and in that place we just call `reloadShoertcutsBar`